case-server/src/main/java/com/powsybl/caseserver/repository/CaseMetadataEntity.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.caseserver.repository;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import lombok.*;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "caseMetadata")
+public class CaseMetadataEntity {
+
+    @Id
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "creationDate", nullable = false)
+    private LocalDateTime date;
```

Comment:
Rename by dateCreation so as not to confuse with the date of the case

Reply:
field removed